src/pages/researcher_console/DatasetSubmissionsTable.jsx

```diff
@@ -65,11 +68,49 @@ export default function DatasetSubmissionsTable(props) {
   ];
 
   const [terms, setTerms] = useState([]);
+  const [selectedTerm, setSelectedTerm] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [rows, setRows] = useState([]);
+  const [open, setOpen] = useState(false);
+
+  const handleClick = (term) => {
+    setOpen(true);
+    setSelectedTerm(term);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const removeDataset = async (termToDelete) => {
+    const termName = termToDelete.datasetName;
+    const termId = termToDelete.datasetId;
+    let updatedTerm = await DataSet.deleteDataset(termId).then(() => {
+        window.location.reload();
+    });
```

Comment:
This causes a refresh while the dialog is open which is slightly confusing as a user - I wasn't clear on what was happening.

Reply:
In the error case, this doesn't close the dialog - it remains open. The error case happens when a dataset is in use. Look for the `deletable` property on the dataset. On the back end, we query the DAR-Dataset join table to see if the dataset has been requested. If it is in use, we mark this field as false.